spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -764,6 +803,19 @@ def __init__(self, parent=None, background_color=None):
         self.fmt = None
         self.fwidth, self.fheight = 200, 200
 
+        self.setContextMenuPolicy(Qt.CustomContextMenu)
+        self.customContextMenuRequested.connect(self.context_menu_requested)
+
+    def context_menu_requested(self, event):
+        """Popup context menu."""
+        if self.fig:
+            pos = QPoint(event.x(), event.y())
+            context_menu = QMenu(self)
+            context_menu.addAction(ima.icon('editcopy'), "Copy Image",
+                                   lambda: copy_figure(self.fig, self.fmt),
+                                   QKeySequence("Ctrl+C"))
```

Comment:
Another hardcoded Ctrl-C

Reply:
I'll check how to implement the shortcut in spyder and apply it.

Reply:
:+1: